addons/main/stringtable.xml

```diff
@@ -919,5 +928,101 @@
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_5">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 5</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_6">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 6</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_7">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 7</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_8">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 8</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_9">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 9</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_10">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 10</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_11">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 11</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_12">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 12</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_13">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 13</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_14">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 14</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_15">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 15</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_16">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 16</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_17">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 17</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_18">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 18</English>
+        </Key>
+        <Key ID="STR_diw_armor_plates_main_autoinjectorItemUsed_Desc_Short_19">
+            <English>Auto-Injector for reducing trauma to the body. Remaining uses: 19</English>
+        </Key>
```

Comment:
look into the sqf command `format` the number at the end can be dynamic.

Reply:
This is for config strings, tried a bunch of ways with macros and couldn't get it to work.